WebExample/package.json

```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "@expo/webpack-config": "^18.0.0",
+    "@react-native-community/slider": "^4.4.2",
```

Comment:
according to https://docs.expo.dev/versions/latest/sdk/slider/, slider doesn't work on web so there's no point adding it as a dependency in WebExample I think

Reply:
Nope, that's not right. It works on the web just fine
<img width="549" alt="image" src="https://user-images.githubusercontent.com/39658211/236785135-89c97ad9-b272-444f-8771-c964822b6deb.png">
<img width="796" alt="image" src="https://user-images.githubusercontent.com/39658211/236785059-c488a98e-3861-4b1e-8da7-44247b309e84.png">